src/api/app/views/webui2/webui/project/_pulse_list_builds.html.haml

```diff
@@ -0,0 +1,23 @@
+- if @builds.any?
+  - builds = @builds.group(:event_type).count
+  There
+  - # FIXME: 'has' does not have an inflection and
+  - # 'has'.pluralize(builds.values.sum, plural: 'have')
+  - # does not work...
+  - if builds.values.sum > 1
```

Comment:
[nitpick] I would write this in one line: `builds.values.sum > 1 ? 'have' : 'has'